ecosystem/indexer/src/schema.rs

```diff
@@ -67,7 +69,7 @@ table! {
 }
 
 table! {
-    write_set_changes (hash) {
+    write_set_changes (transaction_hash, hash) {
```

Comment:
@CapCap I re-ran the diesel migrations, and this change popped up. Hope it's not a concern.